library/src/main/java/com/datatorrent/lib/io/fs/AbstractFileInputOperator.java

```diff
@@ -124,6 +124,34 @@
   protected transient long skipCount = 0;
   private transient OperatorContext context;
 
+  /**
+   * The time between sending batch tuples. Default value is 1 hour. A batch
+   * interval could consist of multiple scan intervals
+   */
+  private long batchIntervalMillis = 60 * 60 * 1000; //1 hour
```

Comment:
Do you have a use case for this? I would think that batch boundary is determined by input, not by processing time. Where are the design aspects documented? The JIRA doesn't have any information.